setup.cfg

```diff
@@ -271,7 +274,6 @@ omit =
   sunpy/cython_version*
   sunpy/*setup*
   sunpy/extern/*
-  sunpy/*/tests/*
```

Comment:
Why delete these lines?

Reply:
We should be checking our test coverage of our tests to ensure we don't keep finding tests we are incorrectly asserting.